docs/current-1.1/contributing.md

```diff
@@ -0,0 +1,63 @@
+# Using the current-1.1 folder and sub-folders
+
+This folder is the top-level node that matches the [docs](../docs)
+folder, but contains deltas for the the .NET Core 1.1 release.
+
+The goal of this separate parallel folder structure is to provide a location
+for the 1.1 release related content that can be relatively easily merged into
+the main structure when provide version switching in the published site.
+
+The content under this node should be a smaller document set that represents
+the deltas from the current Long Term Support (LTS) release and the latest
+fast track release. 
+
+## Structure
+
+There are two cases to adding new content for this release:
+
+* Changes to existing documents
+> Copy the existing content into a parallel folder under this
+> structure. Make your changes, and add the modified file to the
+> TOC for the 1.1 preview release.
+* New documents
+> Put the new document in the proper location, and add it to the TOC
+> under the node for the 1.1 preview release. 
```

Comment:
Why is this attempting to use blockquotes? In any case, it doesn't render properly on GitHub (it renders as actual ">"s).